### PR TITLE
fix error message on creating template

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/page/NodeFormView.coffee [hunk 1]
```diff
@@ -4,7 +4,6 @@ NodeFormView = OrchestraModalView.extend(
     if @options.submitted
       displayRoute = appRouter.generateUrl "showNode",
               nodeId: $('#oo_node_nodeId', @$el).val()
-      Backbone.history.loadUrl(displayRoute)
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/page/NodeFormView.coffee]: if you remove this line, you are redirect to the new node ?

Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/page/NodeFormView.coffee]: yes. displayMenu redirect if first parameter set. That's why i remove this line. The error was on the DOUBLON redirect

**Comment by itkg-alavieille** [comment]
:+1:

**Comment by Thiblef** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**